src/Interval.hs

```diff
@@ -69,94 +69,94 @@ interval <-> x =  interval <+> (-x)
 
 
 raisePerfect :: IQuality -> IQuality
-raisePerfect IPerfect = (IAugmented 1)
-raisePerfect (IAugmented x) = (IAugmented (x+1))
+raisePerfect IPerfect = IAugmented 1
+raisePerfect (IAugmented x) = IAugmented $ x + 1
 raisePerfect (IDiminished 1) = IPerfect
-raisePerfect (IDiminished x) = (IDiminished (x-1)) 
+raisePerfect (IDiminished x) = IDiminished $ x - 1
 
 raiseMajor :: IQuality -> IQuality
-raiseMajor IMajor = (IAugmented 1)
-raiseMajor (IAugmented x) = (IAugmented (x+1))
+raiseMajor IMajor = IAugmented 1
+raiseMajor (IAugmented x) = IAugmented $ x + 1
 raiseMajor IMinor = IMajor
 raiseMajor (IDiminished 1) = IMinor
-raiseMajor (IDiminished x) = (IDiminished (x-1))
+raiseMajor (IDiminished x) = IDiminished $ x - 1
 
 lowerPerfect :: IQuality -> IQuality
-lowerPerfect IPerfect = (IDiminished 1)
-lowerPerfect (IDiminished x) = (IDiminished (x+1)) 
+lowerPerfect IPerfect = IDiminished 1
+lowerPerfect (IDiminished x) = IDiminished $ x + 1
 lowerPerfect (IAugmented 1) = IPerfect
-lowerPerfect (IAugmented x) = (IAugmented (x-1))
+lowerPerfect (IAugmented x) = IAugmented $ x-1
 
 lowerMajor :: IQuality -> IQuality
 lowerMajor IMajor = IMinor
-lowerMajor IMinor = (IDiminished 1)
-lowerMajor (IDiminished x) = (IDiminished (x+1))
+lowerMajor IMinor = IDiminished 1
+lowerMajor (IDiminished x) = IDiminished $ x + 1
 lowerMajor (IAugmented 1) = IMajor
-lowerMajor (IAugmented x) = (IAugmented (x-1))
+lowerMajor (IAugmented x) = IAugmented $ x - 1
 
 
 defaultIQuality :: Int -> IQuality
 defaultIQuality i =
-  case (intMod i) of
+  case intMod i of
```

Comment:
Since the syntactic form `case <e> of ...` expects a single expression, it's never ambiguous; parentheses are never required around this expression.